docs/v1.0/out_elasticsearch.txt

```diff
@@ -139,6 +139,17 @@ For example, td-agent currently bundles the 6.x series of the [elasticsearch-rub
     # For standalone Fluentd users
     $ fluent-gem list elasticsearch
 
+Or, fluent-plugin-elasticsearch v2.11.8 or later, users can inspect version incompatibility with the `validate_client_version` option:
```

Comment:
Is the version number correct? The following commit log suggests that
this particular feature became available in v2.11.7:

https://github.com/uken/fluent-plugin-elasticsearch/commit/f3cfeee29153b133c96d6ed46653e5d748127724